libs/add-child/feature/src/lib/add-child.component.scss

```diff
@@ -0,0 +1,202 @@
+ion-content{
+    --ion-background-color:#EFEFEF;
+}
+
+#nav{
+    display: block;
+    width: 100%;
+    height: 60px;
+    background-color: #176560;
```

Comment:
Better to use ion colours. In this case it would be color(--ion-color-secondary)